clib/mininet_test_base.py

```diff
@@ -2356,7 +2357,7 @@ def start_wpasupplicant(self, host, wpasupplicant_conf, timeout=10, log_prefix='
         wpasupplicant_cmd = mininet_test_util.timeout_cmd(
             'wpa_supplicant -dd -t -c %s -i %s -D wired -f %s %s &' % (
                 wpasupplicant_conf_file_name, host.defaultIntf(), wpasupplicant_log,
-                wpa_ctrl_socket), timeout * 3)
+                wpa_ctrl_socket), 300)
```

Comment:
Should make this a static value / default value 'cause it looks like it's being used a number of times as the default:
(I may be wrong but this is what I'm seeing)
 
https://github.com/faucetsdn/faucet/pull/3124/files#diff-d1e19fafff0f02cf9044fb528163deccR2360

https://github.com/faucetsdn/faucet/pull/3124/files#diff-0b75ebeb379caf55a59782a324483c70R301
https://github.com/faucetsdn/faucet/pull/3124/files#diff-0b75ebeb379caf55a59782a324483c70R306
https://github.com/faucetsdn/faucet/pull/3124/files#diff-0b75ebeb379caf55a59782a324483c70R311
https://github.com/faucetsdn/faucet/pull/3124/files#diff-0b75ebeb379caf55a59782a324483c70R321
https://github.com/faucetsdn/faucet/pull/3124/files#diff-0b75ebeb379caf55a59782a324483c70R609